src/lib/api/index.ts

```diff
@@ -247,6 +251,52 @@ export async function post(agent: BskyAgent, opts: PostOpts) {
       throw e
     }
   }
+
+  try {
+    // TODO: this needs to be batch-created with the post!
```

Comment:
Todo?

Reply:
ah hell, yeah this can wait but it needs to be on our near-term todos. the wonky error message you commented on won't be necessary afterwards